traffic_ops/traffic_ops_golang/servercheck/servercheck.go

```diff
@@ -205,57 +208,79 @@ func ReadServersChecks(w http.ResponseWriter, r *http.Request) {
 	}
 	defer inf.Close()
 
+	data, userErr, sysErr, errCode := handleReadServerCheck(inf, tx)
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, tx, errCode, userErr, sysErr)
+	}
+
+	api.WriteResp(w, r, data)
+}
+
+// DeprecatedReadServersChecks is the handler for deprecated GET requests for /servers/checks
+func DeprecatedReadServersChecks(w http.ResponseWriter, r *http.Request) {
+	inf, userErr, sysErr, errCode := api.NewInfo(r, nil, nil)
+	tx := inf.Tx.Tx
+	if userErr != nil || sysErr != nil {
+		api.HandleDeprecatedErr(w, r, tx, errCode, userErr, sysErr, util.StrPtr(ServerCheck_Get_Endpoint))
```

Comment:
instead of calling `util.StrPtr` couldn't you just give it `&ServerCheck_Get_Endpoint`?

Reply:
Go doesnt let you do & to a const.  so i went this route to keep it a const, but im also open to changing it to a var and then doing &.

Reply:
oh, right. I always forget about that, it's so weird.